source/documentation/continuous-integration-and-delivery/tenancy.md

```diff
@@ -0,0 +1,26 @@
+## Tenancy model
+
+The Reliability Engineering team's Concourse is multi-tenanted:
+
+- Concourse is configured with many teams
+
+- Each teams workloads run on separate infrastructure
+
+- Users in one team cannot interact with the pipelines of another team
+  - (unless the user is a member of both teams)
+
+By default, all Concourse workers share the same egress IP addresses, unless configured otherwise.
+If your team needs separate egress IP addresses, then please ask reliability engineering to make the configuration changes.
```

Comment:
RE should be capitalised if we are being picky

Reply:
Done in a8de5f680fe4b44533c26ebef3b5941f714ab90f